### ISSUE TITLE
Incorrect Substrate tag in tutorial.

### ISSUE THREAD

**Issue by riusricardo** [substrate-developer-hub/substrate-docs#1258]
### Is there an existing issue?

- [X] I have searched the existing issues

### Experiencing problems? Have you tried our Stack Exchange first?

- [X] This is not a support question.

### Bug report for compiling, code snippets, templates, etc.

The Contracts pallet tutorial is pointing to an incorrect tag on the Substrate repo.

The cargo TOML file is showing:
```
...
git = 'https://github.com/paritytech/substrate.git'
tag = 'latest'
...
```
But there is no `latest` tag in the repo. https://github.com/paritytech/substrate/tree/latest
It should be pointing to a release branch.
```
...
git = 'https://github.com/paritytech/substrate.git'
branch = 'polkadot-v0.9.26'
...
```

### Steps to reproduce the problem

https://substrate.stackexchange.com/questions/3804/failed-to-find-tag-latest-pallet-version-in-tutorial

**Comment by lisa-parity** [comment]
Hi @riusricardo, do you know which tutorial still had this tag? It might be that this has been fixed (or that I'm blind 😄 )

**Comment by viac92** [comment]
Another tag fix #1284 PR

**Comment by NukeManDan** [comment]
This has been implemented [github.com/substrate-developer-hub/substrate-docs/pull/1264](https://github.com/substrate-developer-hub/substrate-docs/pull/1264)
